### PR TITLE
Readme Update to use AvatarCreator wording

### PR DESCRIPTION
**Pull request by HarrisonHough**
Small wording and title updates

### CODE REVIEW

**Review by GevorgRPM — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Unreal Engine 5 Web View Example
+# Example UE AvatarCreator Integration
```
Comment (bfercher) [inline README.md:1]: UE Avatar Creator Integration Example

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-This repository contains an Unreal Engine 5 sample project that uses the Unreal Engine Web Browser module (and Web Browser Widget) to embed the Ready Player Me character creator inside an application to create a seamless integration.
+This repository contains an Unreal Engine 5 sample project that uses the Unreal Engine Web Browser module (and Web Browser Widget) to embed the Ready Player Me AvatarCreator inside an application to create a seamless integration.
```
Comment (bfercher) [inline README.md:3]: please use a blank between Avatar and Creator

File: README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ This repository contains an Unreal Engine 5 sample project that uses the Unreal
 
 # Web Browser Demo Map
 
-The WebBrowserDemo map located in the `Content` folder is a good starting point for anybody wanting to use the Web Browser module to seamlessly integrate our RPM web platform into you game or application.  The logic for this is rather simple, if you open up the Level Blueprint you will see this network of nodes.
+The WebBrowserDemo map located in the `Content` folder is a good starting point for anybody wanting to use the Web Browser module to seamlessly integrate our RPM AvatarCreator into you game or application.  The logic for this is rather simple, if you open up the Level Blueprint you will see this network of nodes.
```
Comment (bfercher) [inline README.md:14]: Please use a blank between Avatar and Creator

**Review by bfercher — CHANGES_REQUESTED** [review]
Can you please use Avatar Creator instead of AvatarCreator. And slightly change heading 🙏

File: README.md (hunk 3 above)
Comment (HarrisonHough) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (HarrisonHough) [reply README.md]: done

**Review by bfercher — APPROVED** [review]
🥇

**Pull request merged**